modal/cli/entry_point.py

```diff
@@ -92,6 +93,8 @@ async def setup(profile: Optional[str] = None):
 # Deployments
 entrypoint_cli_typer.add_typer(app_cli, rich_help_panel="Deployments")
 entrypoint_cli_typer.add_typer(container_cli, rich_help_panel="Deployments")
+# TODO: cluster is hidden while multi-node is in beta/experimental
```

Comment:
I was interested in having some config-based way to enable experimental functionality, like `experimental=true` in `~/.modal.toml`. But this didn't seem trivial so I skipped if for this change set.